docs/content/doc/advanced/hacking-on-gitea.en-us.md

```diff
@@ -128,10 +128,28 @@ make revive vet misspell-check
 
 ### Working on JS and CSS
 
-Edit files in `web_src` and run the linter and build the files in `public`:
+For simple changes, edit files in `web_src`, run the build and start the server to test:
 
 ```bash
-make webpack
+make build && ./gitea
+```
+
+For more involved changes use the `watch-frontend` task to continuously rebuild files when their sources change. First, build and run the backend:
```

Comment:
I'd add a suggestion about the `bindata` tag. I think it should not be present?

Reply:
Yes indeed, it must not be present. I initially thought not mentioning it is fine but it's better t be explicit about it.